transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -371,6 +368,10 @@ private void processReady(EpollEventArray events, int ready) {
                 // consume wakeup event
                 Native.eventFdRead(eventFd.intValue());
             } else {
+                if (fd == timerFd.intValue()) {
+                    continue;
```

Comment:
note we don't really care how many timeout events fired so I'm not using `timerfd_gettime`